journal/original/26-chapter-23.adoc

```diff
@@ -126,7 +126,7 @@ in which his Lord would appear unto him, to open the eyes of mankind,
 to turn men from darkness to light, and from the power of Satan unto God.
 That Christ was the Light, the true Light,
 that lighteth every man that cometh into the world; unto whom all mankind must be turned,
-and must come, if they are saved, etc. etc. It was a highly favoured, solemn meeting,
+and must come, if they are saved, etc. It was a highly favoured, solemn meeting,
```

Comment:
After conferring with Jason, I think we think that we should _leave_ this doubled here in the original, but change it in the modernized, so let's switch it back, just here. 👍